neo4j/ci/neo4j.rake

```diff
@@ -0,0 +1,80 @@
+require 'ci/common'
+
+def neo4j_version
+  ENV['FLAVOR_VERSION'] || 'latest'
+end
+
+def neo4j_rootdir
+  "#{ENV['INTEGRATIONS_DIR']}/data_#{neo4j_version}"
+end
+
+container_name = 'dd-test-neo4j'
+container_port = 13306
+
+namespace :ci do
+  namespace :neo4j do |flavor|
+    task before_install: ['ci:common:before_install'] do
+      `docker kill $(docker ps -q --filter name=dd-test-neo4j) || true`
+      `docker rm $(docker ps -aq --filter name=dd-test-neo4j) || true`
+    end
+
+    task install: ['ci:common:install'] do
+      use_venv = in_venv
+      install_requirements("--cache-dir #{ENV['PIP_CACHE']}",
+                           "#{ENV['VOLATILE_DIR']}/ci.log", use_venv)
+      sh %(docker run --name #{container_name} --publish=7474:7474 --publish=7687:7687 --volume=$HOME/neo4j/data:/data --volume=$HOME/neo4j/logs:/logs neo4j:3.1.1 )
+    end
+
+    task before_script: ['ci:common:before_script'] do
+      Wait.for 7474
+      count = 0
+      logs = `docker logs dd-test-neo4j 2>&1`
+      puts "Waiting for Neo4j to come up"
+      until count == 60 or logs.include? 'Remote interface available at'
+        sleep_for 2
+        logs = `docker logs dd-test-neo4j 2>&1`
+        count += 1
+      end
+      if logs.include? 'Remote interface available at'
```

Comment:
From rubocop and linting:

C: Favor modifier if usage when having a single-line body. Another good alternative is the usage of control flow &&/||.